src/jwe.c

```diff
@@ -735,7 +741,12 @@ static bool _cjose_jwe_encrypt_dat_a256gcm(
     }
 
     // get the GCM-mode authentication tag
-    if (EVP_CIPHER_CTX_ctrl(ctx, EVP_CTRL_GCM_GET_TAG, 
+    if (EVP_CIPHER_CTX_ctrl(ctx,
+#if (CJOSE_OPENSSL_11X)
```

Comment:
this would be better managed by using a local variable that's set based on the CJOSE_OPENSSL_11X define, instead of #ifdef-ing within the function call